ngboost/distns/normal.py

```diff
@@ -55,11 +55,6 @@ def fisher_info(self):
         FI[:, 1, 1] = 2
         return FI
 
-    def fisher_info_cens(self, T):
```

Comment:
Taking out since we don't need the censored Fisher information at the moment, but let me know if you'd like to keep that in.

Reply:
sounds good!